templates/zerver/emails/password_reset.subject

```diff
@@ -1 +1,3 @@
-Zulip password reset
+{% if attempted_realm %} Failed password reset attempt
+{% else %} Somebody requested a new password for your Zulip account
+{% endif %}
```

Comment:
I don't understand this change; both of these seem like weird subject links for a password reset email.